src/lib/mina_base/account.ml

```diff
@@ -837,6 +842,29 @@ let has_permission ~control ~to_ (account : t) =
       Permissions.Auth_required.check account.permissions.receive control
   | `Set_delegate ->
       Permissions.Auth_required.check account.permissions.set_delegate control
+  | `Increment_nonce ->
+      Permissions.Auth_required.check account.permissions.increment_nonce
+        control
+
+(** [true] iff account has permissions set that enable them to transfer Mina (assuming the command is signed) *)
```

Comment:
```suggestion
(** [true] if account has permissions set that enable them to transfer Mina (assuming the command is signed) *)
```

Reply:
maybe there is an "iff" that I don't understand? if so, move along and ignore

(and have a great day!)